### PR TITLE
Remove whiteline and update README.

### PR DESCRIPTION
**Pull request by ngzax**
This is the same mod I made to the urbit/examples repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,44 @@
 String processing library for Urbit.
 
 Work in progress, feedback and pull requests welcome!
+
+### Installing / Using
+
+1. Copy lib/string.hoon into {pier}/{desk}/lib
+1. Copy app/string-tester.hoon into {pier}/{desk}/app
```
Comment (Fang-) [inline README.md]: Just `cp -r ./* pier/desk/`? I know that currently breaks on the readme, but urbit/arvo#1121 will likely force that to be fixed.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,44 @@
...
+%delete-last
+%delete-nth
+%delete-all
+%replace
+%replace-first
+%replace-last
+%replace-nth
+%replace-all
+%split
+%glue
```
Comment (Fang-) [inline README.md]: You're fine summarizing these into `%a-bunch-of-test-names` or something, for brevity.

**Review by Fang- — CHANGES_REQUESTED** [review]
Neat, thanks! Here I go nitpicking again. (:

File: README.md (hunk 2 above)
Comment (ngzax) [reply README.md]: Yeah, I actually thought that, but the README isn't that big and this shows them exactly what they'll see so they'll know they did it right. Reasonable?

File: README.md (hunk 1 above)
Comment (ngzax) [reply README.md]: Hah. I was actually going to put a plug in for my https://github.com/ngzax/urbit-sync project which will copy it over and ignore the README but I thought that was too self-serving. :)

If you feel strongly I can change it, but I think it's basically the same and doesn't cause an error now.

File: README.md (hunk 2 above)
Comment (Fang-) [reply README.md]: Eh, you'd also need to keep this updated alongside the code, which is easy to miss.

File: README.md (hunk 1 above)
Comment (Fang-) [reply README.md]: Readme error has been fixed, so we should be good here. (:

Comment (ngzax) [reply README.md]: All right. I'll fix these up and add a new commit today. 😄

**Review by Fang- — APPROVED** [review]
Neat, thank you! (:

**Pull request merged**